agent/bin/ci.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Please add docs explaining what this script does and the expected environment it should run from. E.g it expects env a and b to be set, or if it has to be run from a certain location.

Reply:
Done.